chapters/synchronous.tex

```diff
@@ -201,20 +201,35 @@ \subsection{Clocks and Clocked Variables}\label{clocks-and-clocked-variables}
   }\label{fig:clocked-variable}
 \end{figure}
 
-\subsection{Base-Clock and Sub-Clock Partitions}\label{base-clock-and-sub-clock-partitions}
+\subsection{Base- and Sub-Partitions}\label{base-clock-and-sub-clock-partitions}
 
 There are two kinds of \firstuse[clock!partition]{clock partitions}:
 
-\begin{definition}[Base-clock partition]\index{base-clock partition}\index{clock!partition!base-clock}
-A base-clock partition identifies a set of equations and a set of variables which must be executed together in one task.
-Different base-clock partitions can be associated to separate tasks for asynchronous execution.
+\begin{definition}[Base-partition]\index{base-partition}\index{clock!partition!base-}
+A base-partition identifies a set of equations and a set of variables which must be executed together in one task.
+Different base partitions can be associated to separate tasks for asynchronous execution.
 \end{definition}
 
-\begin{definition}[Sub-clock partition]\index{sub-clock partition}\index{clock!partition!sub-clock}
-A sub-clock partition identifies a subset of equations and a subset of variables of a base-clock partition which are partially synchronized with other sub-clock partitions of the same base-clock
+\begin{definition}[Sub-partition]\index{sub-partition}\index{clock!partition!sub-}
+A sub-partition identifies a subset of equations and a subset of variables of a base-partition which are partially synchronized with other sub-partitions of the same base
 partition, i.e., synchronized when the ticks of the respective clocks are simultaneous.
 \end{definition}
 
+The terminology for the partitions are as follows:
```

Comment:
_terminology_ is singular:
```suggestion
The terminology for the partitions is as follows:
```

Reply:
Some claim that the plural can also be called "terminology"; but here it is a singular use.